### PR TITLE
Named storages option

### PR DESCRIPTION
**Pull request by jbartadev**
Added options to available named storages (`dataset`, `key-value store`, `request queue`)

### CODE REVIEW

**Review by mnmkng — COMMENTED** [review]
Looks good, thanks. Two things:

- please add the corresponding input schema options with tooltips
- please double check that empty fields will be sent as `undefined` and not empty strings, because it could mess up the storage creation in `openDataset(name)` functions. We need the default storages to be used, if the named ones are not provided.

**Comment by jbartadev** [comment]
> Looks good, thanks. Two things:
> 
> * please add the corresponding input schema options with tooltips
> * please double check that empty fields will be sent as `undefined` and not empty strings, because it could mess up the storage creation in `openDataset(name)` functions. We need the default storages to be used, if the named ones are not provided.

Added input schema options. The empty fields will be sent as `undefined` because each of them is optional and it optional string input with no value is not part of the input object.

File: cheerio-scraper/INPUT_SCHEMA.json [hunk 1]
```diff
@@ -197,6 +197,24 @@
 			"default": {},
 			"prefill": {},
 			"editor": "json"
+		},
+		"datasetName": {
+			"title": "Dataset Name",
```
Comment (mnmkng) [inline cheerio-scraper/INPUT_SCHEMA.json:202]: Only capitalize the first letter. => `Dataset name`. Same in the other storages.

File: cheerio-scraper/INPUT_SCHEMA.json [hunk 2]
```diff
@@ -197,6 +197,24 @@
...
+			"type": "string",
+			"description": "Name or ID of the dataset that will be used for storing results.",
```
Comment (mnmkng) [inline cheerio-scraper/INPUT_SCHEMA.json:204]: I would add: 

Name or ID of the dataset that will be used for storing results. If left empty, the default dataset of the run will be used.

File: cheerio-scraper/INPUT_SCHEMA.json [hunk 3]
```diff
@@ -197,6 +197,24 @@
...
+			"editor": "textfield"
+		},
+		"keyValueStoreName": {
+			"title": "Key-value Store Name",
+			"type": "string",
+			"description": "Name or ID of the Key-value store that will be used for storing records.",
+			"editor": "textfield"
+		},
+		"RequestQueueName": {
```
Comment (mnmkng) [inline cheerio-scraper/INPUT_SCHEMA.json:213]: Capital R in `RequestQueueName` => `requestQueueName`

**Review by mnmkng — APPROVED** [review]

**Pull request merged**